src/layer/yolov3detectionoutput.h

```diff
@@ -39,6 +39,19 @@ class Yolov3DetectionOutput : public Layer
     Mat anchors_scale;
     int mask_group_num;
     ncnn::Layer* softmax;
+
+public:
```

Comment:
```protected``` for utility functions

Reply:
These function will be used by child classes.